ramanujan/LHSHashTable.py

```diff
@@ -0,0 +1,233 @@
+import sympy
+import struct
+import mpmath
+import os
+import pickle
+import itertools
+from sympy import lambdify
```

Comment:
redundant...
you can open the project with pycharm and it will show you all the PEP errors / warning 
or with pep8 package